src/core/CLucene/index/FieldInfos.cpp

```diff
@@ -114,6 +118,17 @@ IndexVersion FieldInfos::getIndexVersion() {
 	return IndexVersion::kV0;
 }
 
+uint32_t FieldInfos::getFlags() {
+	size_t numFields = byNumber.size();
+	for (int i = 0; i < numFields; i++) {
+		FieldInfo* fi = fieldInfo(i);
+		if (fi->getFlags() > 0) {
+			return fi->getFlags();
```

Comment:
Is it right to just return the first non-zero flag?

Reply:
May be we can just get fieldInfo(0)?